content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst

```diff
@@ -79,41 +83,181 @@ Reordering Rules`, then click :guilabel:`Create`, and fill out the new line as f
+#. :ref:`visibility days <inventory/product_management/visibility-days>`
+
+.. _inventory/product_management/trigger:
+
+Trigger
+=======
+
+When stock falls below the reordering rule's minimum, set the reordering rule's *trigger* to
+*automatic* to automatically create purchase or manufacturing orders to replenish stock.
 
-Once a reordering rule has been created for a product, if its forecasted quantity is below the
-rule's minimum quantity when the scheduler runs, a replenishment order for the product is
-automatically generated. By default, the scheduler runs once each day.
+On the other hand, setting the reordering rule's trigger to *manual* displays the product and
+forecasted stock on the *replenishment dashboard*, where the procurement manager can review the
+stock levels, lead times, and forecasted dates of arrival.
+
+.. seealso::
+   :doc:`strategies`
 
 .. tip::
-   To manually trigger a reordering rule before the scheduler runs, select :menuselection:`Inventory
-   --> Operations --> Run Scheduler`. Then, select the green :guilabel:`Run Scheduler` button on the
-   pop-up that appears. Be aware that this also triggers any other scheduled actions.
+   The replenishment dashboard is accessed by going to :menuselection:`Inventory app --> Operations
+   --> Replenishment`.
+
+To enable the :guilabel:`Trigger` field, go to :menuselection:`Inventory app --> Configuration -->
+Reordering Rules`. Then, click the :guilabel:`(slider)` icon, and enable the :guilabel:`Trigger`
+option from the additional options drop-down menu.
+
+.. image:: reordering_rules/enable-trigger.png
+   :align: center
+   :alt: Enable the Trigger field by toggling it in the additional options menu
+
+In the :guilabel:`Trigger` column, select :guilabel:`Auto` or :guilabel:`Manual`. Refer to the
+sections below to learn about the different types of reordering rules.
+
+Auto
+----
+
+Automatic reordering rules, configured by setting the reordering rule's :guilabel:`Trigger` field to
+:guilabel:`Auto`, generates purchase or manufacturing orders when:
+
+#. the scheduler runs and the **on-hand quantity** is below the minimum
+#. a sales order is confirmed and lowers the **forecasted quantity** of the product below the
+   minimum
+
+.. tip::
+   The scheduler is set to run once a day by default.
+
+   To manually trigger a reordering rule before the scheduler runs, ensure :ref:`developer mode
+   <developer-mode>` is enabled, and select :menuselection:`Inventory app --> Operations --> Run
+   Scheduler`. Then, click the purple :guilabel:`Run Scheduler` button on the pop-up that appears.
+   Be aware that this also triggers any other scheduled actions.
+
+.. example::
+   The product `Office Lamp` has an automatic reordering rule set to trigger when the forecasted
+   quantity falls below the :guilabel:`Min Quantity` of `5.00`. Since the current forecasted
+   quantity is `55.00`, the reordering rule will not be triggered.
```

Comment:
```suggestion
   The product :guilabel:`Office Lamp` has an automatic reordering rule set to trigger when the
   forecasted quantity falls below the :guilabel:`Min Quantity` of `5.00`. Since the current
   :guilabel:`Forecast` is `55.00`, the reordering rule is **not** be triggered.
```

Reply:
Since Office Lamp is a sample product, I wouldn't put a guilabel here. However, I agree with you that it's better to put guilabels around the Forecast smart button.